docs/src/references/agent/operation-workflow.md

```diff
@@ -500,24 +499,24 @@ This waiting state specifies, with `on_success`, `on_error` and `on_timeout` han
 the next step of the calling workflow on success, failure or timeout of the sub-command.
 
 The `await-command-completion` action also tells how to inject back the outcome of the sub-command into the invoking command state.
-- This is done by attaching values to the `payload` subfields, as in `payload.x.y.z = "${.command.payload.x.y.z}"`
+- This is done by attaching values to `output` subfields, as in `output.x.y.z = "${.payload.x.y.z}"`
 - Many independent subfields can be added. They will all be injected into the current payload of the command state, letting untouched the others.
-- The payload as a whole can be overridden, as in `payload = "${.command.payload}`, however this is discouraged.
+- The invoking command payload cannot be overridden as a whole. Subfields can be added and updated but not removed.
 - Specific path expressions can be used to extract specific excerpts from the sub-command final state.
-- `${.command}` is a json for the whole message including the `topic` and the `payload`.
-- `${.command.topic}` is the sub-command request topic
-- `${.command.topic.target}` is the sub command target identity
-- `${.command.topic.operation}` is the command operation
-- `${.command.topic.cmd_id}` is the command request unique identifier
-- `${.command.payload}` is the whole sub command json payload
-- `${.command.payload.status}` is the sub command final status
-- `${.command.payload.x.y.z}` is the json value extracted from the payload following the given `x.y.z` path if any.
```

Comment:
I also decided to remove the `.command` prefix, even if the data source is the sub-command final state.

- In no case, data will be extracted from several data sources.
- The command init state, i.e. its input, is populated with data extracted from the invoking command state.
- The command final state, i.e. its output, is injected back into the  invoking command state.